wiki/docs/en/learn-docker/bind-mounts.md

```diff
@@ -0,0 +1,53 @@
+Docker isolates all content, code, and data within a container from your local filesystem. Sometimes, however, you may want the container to access a directory on your system. This is where bind mounts come in handy.
```

Comment:
Or to keep persistence from files that are inside the container. For example, you can bind a mount for a database running in a container which enables you to keep the data even if the container is replaced.

Reply:
Never mind I just read persist-data.md which is explaining this.